src/models/Token.js

```diff
@@ -0,0 +1,15 @@
+const { DataTypes } = require('sequelize');
+const { client } = require('../db.js');
+const { User } = require('./User.js');
+
+const Token = client.define('token', {
+  refreshToken: {
+    type: DataTypes.UUID,
```

Comment:
The `refreshToken` field is currently set to `DataTypes.UUID`. If the refresh token is not a UUID format, consider changing the data type to `DataTypes.STRING` to accommodate different token formats.